Feature for apex classes

Aryan-Fexle-May2022 opened the pull request.

Aryan-Fexle-May2022 committed:
Commit-ApexCollections-class

Aryan-Fexle-May2022 committed:
commit-ConstantsExamples

Aryan-Fexle-May2022 committed:
commit-HierarchicalCollectionExample-Class

Aryan-Fexle-May2022 committed:
commit-List-Set-Map-Classes

MittalPriyanka commented on src/classes/HierarchicalCollectionExample.cls:
```diff
@@ -0,0 +1,33 @@
+        
+        //1st Year List Of Students
+        List<student> Year1stListComputer = new List<student>();
+        Year1stListComputer = [SELECT id, Name, Roll_Number__c, Current_Year__c, Section_Stream__c FROM Student__c WHERE Current_Year__c=1 AND Section_Stream__c='Computer Science & Eng.'];
```
fix indentation here

MittalPriyanka commented on src/classes/HierarchicalCollectionExample.cls:
```diff
@@ -0,0 +1,33 @@
+        
+        //1st Year List Of Students
+        List<student> Year1stListElectronic = new List<student>();
+        Year1stListElectronic = [SELECT id, Name, Roll_Number__c, Current_Year__c, Section_Stream__c FROM Student__c WHERE Current_Year__c=1 AND Section_Stream__c='Electrical Engineering'];
```
When ever your line is being too long, give a line break;
eg:
 [SELECT id, Name, Roll_Number__c, Current_Year__c, Section_Stream__c FROM Student__c 
   WHERE Current_Year__c=1 AND Section_Stream__c='Electrical Engineering'];

Aryan-Fexle-May2022 committed:
Updating-HierarchicalCollectionExample-commit

Aryan-Fexle-May2022 committed:
commit-MapHierarchalPractice-class

Aryan-Fexle-May2022 committed:
commit-WorkingWithsObjects-class

Aryan-Fexle-May2022 committed:
Commit-Call-By-Value-And-Reference-Example

Commiting - CallByReferenceExample Class and CallByValueExample Class

Aryan-Fexle-May2022 committed:
Commit-ConditionalStatementPractices-Class

Committing - Conditional Statement(if-else) Example Class

Aryan-Fexle-May2022 committed:
Commit-PurchaseOrderInterface-Example

Committing- PurchaseOrderInterface Interface along with CustomerPurchaseOrderClass class and EmployePurchaseOrderClass class

Aryan-Fexle-May2022 committed:
Commit-LoopPractices-Class

Committing-Loop Example

Aryan-Fexle-May2022 committed:
Commit-MethodOverloadingExample-Class

Committing- Method Overloading Example

Aryan-Fexle-May2022 committed:
Commit-Overidden-And-Overiding-Class-Example

Committing - OveriddenMarkerClass and OveridingYellowMarkerClass

Aryan-Fexle-May2022 committed:
Commit-Final-Keyword-Example

Committing - PracticeFinalKeywordClass class Example

Aryan-Fexle-May2022 committed:
Commit-Date-Class-Example

Aryan-Fexle-May2022 committed:
Commit-Date-Time-Class

Aryan-Fexle-May2022 committed:
Commit-String-Class-Example

Aryan-Fexle-May2022 committed:
Commit-Primitive-Data-Type-Class-Example

Aryan-Fexle-May2022 committed:
Commit-Supper-Class-Example

Here, SuperClass is Parent Class and SubClass is Child Class
